sw/vendor/riscv_compliance/riscv-target/opentitan/README.md

```diff
@@ -112,6 +113,27 @@ OK: 48/48
 There are several test suites that can be run `rv32i`, `rv32im`, `rv32imc` and `rv32Zicsr`.
 Change the `RISCV_ISA` argument passed to `make` to choose between them.
 
+## Changing targets
+When switching between targets (i.e. between FPGA and verilator) the `work` directory in the `riscv_compliance` tree must be remove to force a software rebuild.
```

Comment:
READMEs generally wrap at 80 characters.

Reply:
Indeed, this one didn't though so matched the existing style (one line per sentence). I'll rewrap it all if I do end up making other changes.